examples/msas/predict_with_msas.py

```diff
@@ -46,7 +33,7 @@
     seed=42,
     device="cuda:0",
     use_esm_embeddings=True,
-    msa_directory=msa_dir,
+    msa_directory=Path(__file__).parent,
```

Comment:
No - this example is meant to be for manual MSAs.

The MSA server is finnicky to have an example for because we want to actually operate on the _chains_ that have been parsed not the raw inputs, as this avoids any weirdness with things that failed to tokenize, modified residues, etc. While this example itself doesn't suffer from these issues, we don't want to imply that reading from inputs to make MSAs is a general solution. People can look at the code in `chai1.py` for an example of how that works.

Reply:
Consider the modified residues example: the `input.sequence` field might then contain a sequence like `(MSE)RKDES`. When we pass that string to MMseqs2, it's unclear how that should be handled. By parsing sanitized chain sequences, this automatically becomes `XRKDES`.

Reply:
Yes - passing `msa_server=True` here would handle correctly, but that can also be easily inferred from the CLI interface. I think it's more valuable that we point users to the example `aligned.pqt` files in this folder and show how those can be passed in, compared to showing a boolean flag.

I'll add a comment re the flag though.